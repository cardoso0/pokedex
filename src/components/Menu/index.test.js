import renderer from 'react-test-renderer'
import '@testing-library/jest-dom'
import { BrowserRouter } from 'react-router-dom'

import { Menu } from '.'

const renderMenu = 
  <BrowserRouter>
    <Menu />
  </BrowserRouter>

describe("Menu Component", () => {
  it("component rendering | snapshot", () => {
    const tree = renderer
      .create(renderMenu)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it("Menu component rendering", () => {
    const tree = renderer
      .create(renderMenu)
    expect(tree).toBeTruthy()
  })
})