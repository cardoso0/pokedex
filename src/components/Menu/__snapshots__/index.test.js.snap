// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu Component component rendering | snapshot 1`] = `
<nav
  className="sc-bczRLJ gIEkxD"
>
  <a
    href="/trainers/"
    onClick={[Function]}
  >
    Home
  </a>
  <a
    href="/trainers/pokedex"
    onClick={[Function]}
  >
    Pok√©dex
  </a>
  <a
    href="/trainers/favorites"
    onClick={[Function]}
  >
    Menu.fav
  </a>
</nav>
`;
